package.json

```diff
@@ -33,6 +33,7 @@
 		"eslint": "^6.8.0",
 		"eslint-plugin-eslint-comments": "3.1.2",
 		"eslint-plugin-import": "2.18.2",
+		"eslint-plugin-jest": "^25.3.0",
```

Comment:
@SiobhyB is this dependency upgrade required for this feature? Thanks!

Reply:
It's not, no, thanks for spotting that! I had some trouble getting tests to pass with this PR, so kept running `npm install` and this was a change that got committed as part of that. I've updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4352/commits/79e5caf1ae84111d2c1157f67b3754c685af04e4.